tests/ui/features/@vulnerability-explorer/vulnerability-explorer.step.ts

```diff
@@ -7,6 +7,7 @@ export const { Given, When, Then } = createBdd();
 
 const SBOM_TABLE_NAME = "Sbom table";
 const ADVISORY_TABLE_NAME = "Advisory table";
+const COLUMN_LABELS = ["ID", "Title", "Type", "Revision", "Vulnerabilities"];
```

Comment:
Suggestion: Same as on [PR#67](https://github.com/trustification/trustify-tests/pull/67), instead of creating a separate list - this can be handled as part of BDD statement